<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>About</class>
  <widget class="QDialog" name="About">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>400</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>About</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QLabel" name="lbl_Title">
          <property name="font">
            <font>
              <pointsize>15</pointsize>
              <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string>CyberDom</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lbl_Author">
          <property name="text">
            <string>By Desire_Games</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lbl_Version">
          <property name="font">
            <font>
              <pointsize>14</pointsize>
            </font>
          </property>
          <property name="text">
            <string>Version 0.0.1 Alpha</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer_Top">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QGridLayout" name="gridLayout_Info">
          <item row="0" column="0">
            <widget class="QLabel" name="lbl_Script">
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Script:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLabel" name="lbl_ScriptName">
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Unknown</string>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="lbl_SVersion">
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Version:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLabel" name="lbl_SVersionN">
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Unknown</string>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="lbl_Path">
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Path:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QLabel" name="lbl_PathDir">
              <property name="font">
                <font>
                  <pointsize>12</pointsize>
                </font>
              </property>
              <property name="text">
                <string>TextLabel</string>
              </property>
              <property name="wordWrap">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="verticalSpacer_Bottom">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>40</height>
            </size>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="QLabel" name="lbl_Disclaimer">
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
          <property name="text">
            <string>CyberDom is freeware, use at your own risk!</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
          <item>
            <widget class="QPushButton" name="btn_AboutQt">
              <property name="text">
                <string>About Qt</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QDialogButtonBox" name="buttonBox">
              <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="standardButtons">
                <set>QDialogButtonBox::StandardButton::Ok</set>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections>
    <connection>
      <sender>buttonBox</sender>
      <signal>accepted()</signal>
      <receiver>About</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>248</x>
          <y>254</y>
        </hint>
        <hint type="destinationlabel">
          <x>157</x>
          <y>274</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>buttonBox</sender>
      <signal>rejected()</signal>
      <receiver>About</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>316</x>
          <y>260</y>
        </hint>
        <hint type="destinationlabel">
          <x>286</x>
          <y>274</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
